sockeye/evaluate.py

```diff
@@ -38,7 +38,8 @@ def raw_corpus_bleu(hypotheses: Iterable[str], references: Iterable[str], offset
     :param offset: Smoothing constant.
     :return: BLEU score as float between 0 and 1.
     """
-    return sacrebleu.raw_corpus_bleu(hypotheses, [references], smooth_floor=offset).score / 100
+    return sacrebleu.raw_corpus_bleu(hypotheses, [references], smooth_floor=offset).score / 100.0
+
```

Comment:
Python 3 is supposed to be robust to this, right? `//` is for explicitly integer division. This is fine for clarity but it shouldn't have any effect.

Reply:
Sure that doesn't make a difference, just for clarity.